src/main/webapp/i18n/de/exerciseChatbot.json

```diff
@@ -26,7 +26,7 @@
             "text4_3": "<b>Vergrößern und Verkleinern des Widgets:</b> Klicke und ziehe die Ränder des Widgets in verschiedene Richtungen, um die Größe zu ändern.",
             "text4_4": "<b>Vollbildmodus:</b> Du kannst das Widget im Vollbildmodus öffnen, indem du auf den Vollbildmodus-Button (in der oberen rechten Ecke des Widgets) klickst.",
             "text4_5": "<b>Chatverlauf löschen:</b> Du kannst deinen Chatverlauf löschen, indem du auf den Mülleimer klickst (oben rechts im Widget). Dadurch werden alle vorherigen Nachrichten gelöscht und du kannst einen neuen Chat starten.",
-            "supportText": "(Wenn du Probleme mit dem Iris-Widget oder bei der Interaktion mit Iris hast, erstelle bitte einen Issue auf GitHub https://github.com/ls1intum/Artemis)",
+            "supportText": "(Wenn du Probleme mit dem Iris-Widget oder bei der Interaktion mit Iris hast, erstelle bitte ein Issue auf GitHub https://github.com/ls1intum/Artemis)",
```

Comment:
Are you sure that tie dot does not become part of the URL?

Reply:
I don’t think so since it does not automatically turn into a link anyway (unless this is in a Markdown field; in that case the common Markdown parsers are usually smart enough to not put the `.` as part of the URL).

Reply:
Added in both languages with 0e1596b81d789216f8805e668d91286a57fbbfc6